drivers/counter/rtc_mcp7940n.c

```diff
@@ -31,10 +31,16 @@ LOG_MODULE_REGISTER(MCP7940N, CONFIG_COUNTER_LOG_LEVEL);
 /* Largest block size */
 #define MAX_WRITE_SIZE                  (RTC_TIME_REGISTERS_SIZE)
 
-/* tm struct uses years since 1900 but unix time uses years since
- * 1970. MCP7940N default year is '1' so the offset is 69
+/* The MCP7940N RTCC does not keep century bits in its registers.
+ * This uses the user-accessible SRAM to store century bits for the driver.
  */
-#define UNIX_YEAR_OFFSET		69
+#define RTC_CENTURY		0x32
```

Comment:
Any specific reason why the address is `0x32`? Using the predefined `SRAM_MIN` or `SRAM_MAX` registers would explain where it's coming from.

Reply:
It doesn't really matter which address it is as long as its within the user accessible SRAM. I chose an address that is in the middle of this space and defined it. The definition could be moved to the header file if needed. The basis of my change comes from a similar change done on this #59402 driver for a different RTC with a similar address map.